### PR TITLE
action_recognition: fixes

### PR DESCRIPTION
**Pull request by Wovchena**
enable capturing from camera
wait for Esc or q
fix misspelled fps->ms
should be merged after https://github.com/opencv/open_model_zoo/pull/229

### CODE REVIEW

**Comment by IRDonch** [comment]
Rebase, please - the history looks weird otherwise.

File: demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py [hunk 1]
```diff
@@ -86,7 +86,8 @@ def render_frame(self, frame, logits, timers, frame_ind):
 
         cv2.imshow("Action Recognition", frame)
 
-        if cv2.waitKey(1) & 0xFF == ord('q'):
+        key = cv2.waitKey(1) & 0xFF
+        if key == ord('q') or key == ord('Q') or key == 27:
```
Comment (IRDonch) [inline demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py]: You can simplify this to `if key in {...}:`

File: demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def render_frame(self, frame, logits, timers, frame_ind):
         if logits is not None:
             labels, probs = decode_output(logits, self.lables, top_k=self.number_of_predictions,
                                           label_postprocessing=self.postprocessing)
-            print("Frame {}: {} - {:.2f}% -- {:.2f} fps".format(frame_ind, labels[0], probs[0] * 100, inference_time))
+            print("Frame {}: {} - {:.2f}% -- {:.2f}ms".format(frame_ind, labels[0], probs[0] * 100, inference_time))
```
Comment (IRDonch) [inline demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py]: "ms" should still be separated from the number by a space.

File: demos/python_demos/action_recognition/action_recognition.py [hunk 3]
```diff
@@ -41,7 +41,9 @@ def build_argparser():
     args.add_argument("-m_en", "--m_encoder", help="Required. Path to encoder model", required=True, type=str)
     args.add_argument("-m_de", "--m_decoder", help="Required. Path to decoder model", required=True, type=str)
     args.add_argument("-i", "--input",
-                      help="Required. Path to a video or a .txt file with a list of video files (one video per line)", type=str)
+                      help="Required. Id of the video capturing device to open (to open default camera just pass 0), "
```
Comment (IRDonch) [inline demos/python_demos/action_recognition/action_recognition.py]: If it's required, you should add `required=True`.

File: demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py (hunk 2 above)
Comment (Wovchena) [reply demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py]: `Incremental timer` has `"{:.2f}ms (±{:.2f}) {:.2f}fps".format(self.avg, self.std, self.fps)`. I removed a space for consistency.

File: demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py (hunk 1 above)
Comment (Wovchena) [reply demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py]: Updated

File: demos/python_demos/action_recognition/action_recognition.py (hunk 3 above)
Comment (Wovchena) [reply demos/python_demos/action_recognition/action_recognition.py]: Added

Comment (IRDonch) [reply demos/python_demos/action_recognition/action_recognition.py]: This also changes the `--help` output, so update the README correspondingly.

File: demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py (hunk 2 above)
Comment (IRDonch) [reply demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py]: Alright.

File: demos/python_demos/action_recognition/action_recognition.py (hunk 3 above)
Comment (Wovchena) [reply demos/python_demos/action_recognition/action_recognition.py]: The output did not change for me.

Comment (IRDonch) [reply demos/python_demos/action_recognition/action_recognition.py]: Are you sure? I just tried it myself, and the brackets around `-i INPUT` disappear.

Comment (Wovchena) [reply demos/python_demos/action_recognition/action_recognition.py]: Got it. Updated

**Pull request merged**